nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/pom.xml

```diff
@@ -426,5 +426,10 @@
             <artifactId>nifi-mock</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-record-serialization-service-api</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this brought in just for this line?
```
                .identifiesControllerService(RecordReaderFactory.class)
```
If so, I think we can eliminate the dependency and just use:
```
                .identifiesControllerService(ControllerService.class)
```
Would like to avoid bringing an extension's api nar into framework, even for tests, if not necessary.

Reply:
Thanks!  Worked fine.